### ISSUE TITLE
Suggestion: use `pudl_` prefix for all command line scripts

### ISSUE THREAD

**Issue by karldw** [catalyst-cooperative/pudl#460]
**Is your feature request related to a problem? Please describe.**

I forget what the various PUDL commands are, and I'm lazy. I know a bunch start with `pudl_`, but what's the sqlite one called? It would be nice to just type `pudl_` and tab complete.

**Describe the solution you'd like**

It would be convenient if all of the PUDL command-line tools were prefixed with `pudl_`
Here are possible re-names:
* `datapkg_to_sqlite` → `pudl_datapkg_to_sqlite` 
* `epacems_to_parquet` → `pudl_epacems_to_parquet` 
* `ferc1_to_sqlite` → `pudl_ferc1_to_sqlite`

**Describe alternatives you've considered**

Other options would be remembering the commands better, or maybe having a docs page that lists all of the command line tools.

**Comment by zaneselvans** [comment]
Yeah, I've thought about doing this too.  Or using the command/subcommand structure that `conda` and `git` and `apt` and a bunch of other unix tools have. So like `pudl data` and `pudl ferc1_to_sqlite` or something.  But agree, with tab completion, this is unnecessarily complicated as it is.
